forward_engineering/helpers/alterScriptHelpers/alterEntityHelper.js

```diff
@@ -1,5 +1,6 @@
+const _ = require('lodash');
+
 module.exports = (app, options) => {
-	const _ = app.require('lodash');
 	const { getEntityName } = app.require('@hackolade/ddl-fe-utils').general;
 	const { createColumnDefinitionBySchema } = require('./createColumnDefinition')(_);
```

Comment:
can we take an opportunity to reduce the required dependencies across all functions within plugin? 🙂